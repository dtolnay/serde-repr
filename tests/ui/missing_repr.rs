use serde_repr::SerializeRepr;

#[derive(SerializeRepr)]
enum SmallPrime {
    Two = 2,
    Three = 3,
    Five = 5,
    Seven = 7,
}

fn main() {}
