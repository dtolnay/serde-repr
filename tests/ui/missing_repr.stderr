error: missing #[repr(...)] attribute
 --> $DIR/missing_repr.rs:3:10
  |
3 | #[derive(SerializeRepr)]
  |          ^^^^^^^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
